### DISCUSSION TITLE
I can't log in

### DISCUSSION THREAD

**Discussion by arn4** [victorbalssa/abacus#7] [Q&A] [answered]
Hi!
This probably goes to the discussion section, but I can't find it.
Just downloaded the app form the app store, but I can't manage to log in. 
I get "invalid_client" error. I've just created a new token, copied the secret and read the Client ID from the database, but still it does not work. Do you have any suggestion?

Thanks for the app!

**Comment by logdotexe** [comment]
Same issue here

**Comment by victorbalssa** [comment, marked as answer]
# How to connect to your back-end instance: [HELP.md](https://github.com/victorbalssa/abacus/blob/updates/.github/HELP.md)

1) You need to create a Oauth client with redirect URI: `abacusiosapp://redirect`

2) copy paste `Oauth Client ID` it will be a number (required).

3) No need to use the secret client but if you do so copy paste it in the `Oauth Client Secret` field (optional).

**Reply by victorbalssa** [reply]
@arn4 any news?

**Comment by arn4** [comment]
I've realised I had completely misunderstood how the Oauth works. Now I'm able to log in!
Still, I can't add transaction from the app, only see some reports

**Reply by victorbalssa** [reply]
@arn4 Ok good news.
I'm working on this iOS client for FireflyIII as a side project.

Could you create a small list of the most important features you want to have access from your iPhone?

1. Add transactions. (transfer, withdraw, deposit)
2. Access to specific graph.
3. Access to categories details.
4. Face ID lock.
5. ...

If you found bugs or UI problems, you can create an issue on this repo.
Thanks,
Victor

**Comment by realzsan3** [comment]
I have the same problem, I am using https://ip:port/ and the client is up to date, does it not support ip login?

**Reply by realzsan3** [reply]
> I have the same problem, I am using https://ip:port/ and the client is up to date, does it not support ip login?

The problem was found to be with my nginx, so I was able to log in after removing nginx
